src/main/java/org/codehaus/mojo/tidy/TidyMojo.java

```diff
@@ -46,6 +46,14 @@ public abstract class TidyMojo extends AbstractMojo {
     @Parameter(defaultValue = "${project}", required = true, readonly = true)
     protected MavenProject project;
 
+    /**
+     * The path of the pom file to process.
+     *
+     * @since 1.3.0
```

Comment:
Please also update project to version 1.3.0-SNAPSHOT ... we can forget about it during release

Reply:
TBH I didn't know what was the recommended procedure. Done.